src/main/java/com/teamsfinder/userwriteservice/tag/dto/TagMapper.java

```diff
@@ -0,0 +1,18 @@
+package com.teamsfinder.userwriteservice.tag.dto;
+
+import com.teamsfinder.userwriteservice.tag.model.Tag;
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class TagMapper {
+    public static TagResponseDto mapTagToDto(Tag tag){
```

Comment:
daj pod deklaracją klasy linijke odstępu, wygodniej to czytać

Reply:
Konieczne to jest? Musialbym w kazdej klasie zmieniac

Reply:
skonfiguruj sobie porządnie auto formater i przeleć po projekcie po prostu